ironfish/src/network/testUtilities/mockHostsStore.ts

```diff
@@ -51,7 +51,7 @@ class MockFileSystem extends FileSystem {
 
 class MockHostsStore extends HostsStore {
   constructor() {
-    super(new MockFileSystem())
+    super(new MockFileSystem(), "~/.ironfish")
```

Comment:
Small nit, not mandatory but we could just pass DEFAULT_DATA_DIR here from config.ts